input/appendixareport.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
need there be two of the same input file..?

Reply:
I can remove them, i just used the json file to generate the xml file, using the ``` --json-to-xml ``` cyclus command